pod2man.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+pod2man_helper() {
+    local podfile="${1?}"
```

Comment:
Now that this function is simpler, I might be inclined to just drop it, or make it even more of a helper, i.e. just put the actual invocations in the code below, or do something like this:

```sh
gen-page puppet-db 8 "manages PuppetDB administrative tasks" "$mandir"
gen-page puppet-query 8 "queries PuppetDB data" "$mandir"
...
```

where mandir is perhaps .../usr/share/man.